Source/WebCore/css/ComputedStyleExtractor.cpp

```diff
@@ -574,6 +574,36 @@ static RefPtr<CSSValue> positionOffsetValue(const RenderStyle& style, CSSPropert
     return CSSPrimitiveValue::create(CSSValueAuto);
 }
 
+// FIXME: Replace calls to this function with `getCSSPropertyValuesForShorthandProperties`
+// once white-space longhands are completely implemented.
+Ref<CSSValue> ComputedStyleExtractor::getWhiteSpaceValue(const RenderStyle& style)
```

Comment:
```suggestion
Ref<CSSValue> ComputedStyleExtractor::whiteSpacePropertyValue(const RenderStyle& style)
```

WebKit style guide says to not start function names with `get`.